.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -32,7 +32,7 @@ steps:
         mkdir -p ~/.kube/
         echo "Create AKS Overlay cluster"
         make -C ./hack/swift azcfg AZCLI=az REGION=$(REGION_OVERLAY_CLUSTER_TEST)
-        make -C ./hack/swift overlay-no-kube-proxy-up AZCLI=az REGION=$(REGION_OVERLAY_CLUSTER_TEST) SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        make -C ./hack/swift overlay-no-kube-proxy-up AZCLI=az REGION=$(REGION_OVERLAY_CLUSTER_TEST) SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}-$(make revision) VM_SIZE=Standard_DS4_v2
```

Comment:
`Standard_DS4_v2` is a HUGE jump up in VM size and cost (12x!). I originally picked B2s as the cheapest viable option - did you evaluate any other SKUs? Maybe we could start at `Standard_B2ms` which has double the mem at only double the cost?

Reply:
The `Standard_B2ms` worked without the resource limit. @rbtr . Thanks for the help
Will try another run to be sure.